layercode.js

```diff
@@ -0,0 +1,545 @@
+require('jquery');
```

Comment:
I am curious as to why it looks like all the lines were added. May I know the steps you took to make this PR? Thanks!

Reply:
Ah! I see that you created a new layercode.js file and added it to the root. The changes need to be added to the file in the src folder.